src/components/CartPrice.js

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from "react-router-dom";
+import { useProducts } from "../context/products";
+
+export const CartPrice = () => {
+  const navigate = useNavigate();
+  const {
+    productsState: {
+      cartItems,
+      totalPrice,
+      totalDiscount,
+      totalDeliveryCharge,
+    },
+  } = useProducts();
+  return (
+    <div class="bg-white pd-md price-details-cart h-max-content">
+      <h3 class="flex-center">Price Details</h3>
+      <hr />
```

Comment:
I think we should not use `<hr/>` and `<br/>` tags in react. Please check once.

Reply:
I think we can use hr tag, just br tag  we must avoid